src/grpc_interop.cc

```diff
@@ -285,6 +285,36 @@ LIBRARY_EXPORT int32_t RegisterEnumMetadata2(grpc_labview::gRPCid** id, grpc_lab
     return 0;
 }
 
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+LIBRARY_EXPORT int32_t GetLVEnumValueFromProtoValue(grpc_labview::gRPCid** id, const char* enumName, int protoValue)
+{
+    auto server = (*id)->CastTo<grpc_labview::MessageElementMetadataOwner>();
+    if (server == nullptr)
+    {
+        return -1;
+    }
+    auto metadata = (server.get())->FindEnumMetadata(std::string(enumName));
+    return metadata.get()->GetLVEnumValueFromProtoValue(protoValue);
+    
+    //return 0;
```

Comment:
remove comment, as discussed use the error code as return type. for proto value pass int*, on the caller side user error handler subvi.

Reply:
Fixed.